### PR TITLE
Fix the function namespace bug

### PR DESCRIPTION
**Pull request by frag-seb**
The namespace for the functions has changed (with php-pm/php-pm@8a1db9c ) and the composer.lock is not important.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,11 +4,11 @@ HttpKernel adapter for use of Symfony and Laravel frameworks with PHP-PM. See ht
 
 ### Setup
 
-  1. Install PHP-PM
+  1. Install PHP-PM (is not mandatory)
 
         composer require php-pm/php-pm:dev-master
 
-  2. Install HttpKernel Adapter
+  1. Install HttpKernel Adapter
```
Comment (andig) [inline README.md]: Could you undo the changes to this file? Without php-pm this a bit pointless?

File: .gitignore [hunk 2]
```diff
@@ -1 +1,2 @@
-/vendor/
\ No newline at end of file
+/vendor/
+composer.lock
```
Comment (andig) [inline .gitignore]: Could you please undo this unrelated change?

File: README.md (hunk 1 above)
Comment (frag-seb) [reply README.md]: The component php-pm is in the composer.json require and will be reloaded.

File: .gitignore (hunk 2 above)
Comment (frag-seb) [reply .gitignore]: You have more problems in this case with composer.lock than it helps.

example:
composer.lock php-pm last commit: cd12ed86ac4d043d584cb1717a2bae64b576c1a0
php-pm last commit is 70e7b3134e85b30489c3c9e1a670e1ced83afae5

And when you the README.md take seriously it will not work.

File: README.md (hunk 1 above)
Comment (andig) [reply README.md]: Then remove the ordered list?

Comment (frag-seb) [reply README.md]: I do not understand exactly

You mean so:

### Setup

  Install PHP-PM (is not mandatory)

        composer require php-pm/php-pm:dev-master

  Install HttpKernel Adapter

        composer require php-pm/httpkernel-adapter:dev-master

Comment (andig) [reply README.md]: For example, yes. In any case an ordered list with only one entry (1.) does not make much sense :)

Comment (frag-seb) [reply README.md]: Ok I will add 1 and 2 to the list :D

**Comment by andig** [comment]
You're still git-ignoring the lock file but committing a changed version. Can you remove all changes related to the lock file?

**Comment by frag-seb** [comment]
Then my change will not work. Should I undo it defiantly?

**Comment by frag-seb** [comment]
Example for .gitignore 

https://github.com/alcaeus/mongo-php-adapter/blob/master/.gitignore

I thinks it is better that we close this pull request and we open a issues.

**Comment by andig** [comment]
> I thinks it is better that we close this pull request and we open a issues.

We can continue here. What you cannot do is leave composer.lock in the repo for now but exclude if from gitignore for future commits. Just ignore it and remove the file or leave as is.

**Comment by frag-seb** [comment]
I understand it now, I did not commit the deleted composer.lock. I would, if it is ok?

**Comment by frag-seb** [comment]
And I add then the composer.lock to the gitignore file.

**Comment by andig** [comment]
Yes- should be ok now that we have release versions of all dependencies :)

**Comment by frag-seb** [comment]
Can you check please?

**Comment by andig** [comment]
you still need to remove the lock file and please remove the "is not mandatory"- this pacakge without php-pm does not make sense ;)

**Comment by frag-seb** [comment]
Delete Commit: 12dd947744f5f99b6d6bd63783d17f256a24c5cc

>  this pacakge without php-pm does not make sense ;)

The php-pm/php-pm is require in the composer.json and does not need to be loaded but I change it.
commit: 28feb455c90aef5d8841716eaf884a2dcd973643
